.github/workflows/ci-cd-pims-master.yml

```diff
@@ -0,0 +1,120 @@
+name: CI-CD PIMS UAT Development
+env:
+  OPENSHIFT_SERVER: ${{ secrets.OPENSHIFT_SERVER }}
+  # service account: gitaction
+  OPENSHIFT_TOKEN: ${{ secrets.OPENSHIFT_TOKEN }}
+  OPENSHIFT_TOOLS_NAMESPACE: "3cd915-tools"
+  MS_TEAMS_WEBHOOK_BUILD_CHANNEL: ${{ secrets.MS_TEAMS_WEBHOOK_URI_BUILD_CHANNEL }}
+
+  ## variables for scripts under git\openshift\4.0\scripts\oc-*.sh
+  APP_PORT: 8080
+  DESTINATION: "master"
+  OC_JOB_NAME: "master"
+  GIT_URL: "${{github.server_url}}/${{github.repository}}"
+  GIT_BRANCH: "${{github.ref}}"
```

Comment:
Have we tested that github.ref would return "master" when this workflow is run manually? I'd lean towards just hardcoding to "master" - thoughts?